app/routes/questionnaire.py

```diff
@@ -417,6 +452,26 @@ def get_view_submitted_response(schema, questionnaire_store):
     return view_submitted_response.get_rendered_html()
 
 
+@questionnaire_blueprint.route("download-preview-pdf", methods=["GET"])
+@with_questionnaire_store
+@with_schema
+def get_preview_questions_pdf(
+    schema: QuestionnaireSchema, questionnaire_store: QuestionnaireStore
+) -> Response:
+    view_preview_questions_pdf = PreviewQuestionsPDF(
+        schema,
+        questionnaire_store,
+        flask_babel.get_locale().language,
+    )
```

Comment:
Will this not need the try catch from above. This will error if you try to download when preview isn't enabled?.
```py
	try:
        view_preview_questions_pdf = PreviewQuestionsPDF(
	        schema,
	        questionnaire_store,
	        flask_babel.get_locale().language,
	    )
    except PreviewNotEnabledException as exc:
        raise NotFound from exc
```